src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas3/Cas3ReportService.kt

```diff
@@ -162,6 +164,11 @@ class Cas3ReportService(
       }
   }
 
+  fun createBookingGapReport(): String {
```

Comment:
Currently no route for a user to invoke this function.  Only called during integration/unit tests.

Reply:
This is ok for now, the query can be run against the database direct for now until we determine whether a proper report is required.